mlrun/db/httpdb.py

```diff
@@ -205,6 +210,10 @@ def api_call(
             )
 
         try:
+            if stream:
+                return self.session.stream_request(
+                    method, url, to_stdout=to_stdout, timeout=timeout, **kw
+                )
```

Comment:
Does anybody use it (client/sdk/ui)?
I couldn't find a call with `stream=True`.
If not we can remove it